pkgs/development/python-modules/jira/default.nix

```diff
@@ -12,7 +14,7 @@ buildPythonPackage rec {
   };
 
   buildInputs = [ glibcLocales pytest pytestcov pytestrunner pbr ];
-  propagatedBuildInputs = [ requests requests_oauthlib requests_toolbelt defusedxml ];
+  propagatedBuildInputs = [ requests requests_oauthlib requests_toolbelt defusedxml sphinx pbr ];
```

Comment:
Are  `sphinx` and `pbr` really needed at runtime? If they're used only for building, why propagate them?